solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+
+//! The bridge to ethereum config
+
+pub const SLOTS_PER_EPOCH: u32 = snowbridge_pallet_ethereum_client::config::SLOTS_PER_EPOCH as u32;
+use crate::{parameter_types, weights, Runtime, RuntimeEvent};
+use snowbridge_beacon_primitives::{Fork, ForkVersions};
+
+// For tests, benchmarks and fast-runtime configurations we use the mocked fork versions
+#[cfg(any(
+    feature = "std",
+    feature = "fast-runtime",
+    feature = "runtime-benchmarks",
+    test
+))]
+parameter_types! {
+    pub const ChainForkVersions: ForkVersions = ForkVersions {
+        genesis: Fork {
+            version: [0, 0, 0, 0], // 0x00000000
+            epoch: 0,
+        },
+        altair: Fork {
+            version: [1, 0, 0, 0], // 0x01000000
+            epoch: 0,
+        },
+        bellatrix: Fork {
+            version: [2, 0, 0, 0], // 0x02000000
+            epoch: 0,
+        },
+        capella: Fork {
+            version: [3, 0, 0, 0], // 0x03000000
+            epoch: 0,
+        },
+        deneb: Fork {
+            version: [4, 0, 0, 0], // 0x04000000
+            epoch: 0,
+        }
+    };
+}
+
+// Holesky: https://github.com/eth-clients/holesky
+#[cfg(not(any(
+    feature = "std",
+    feature = "fast-runtime",
+    feature = "runtime-benchmarks",
+    test
+)))]
+parameter_types! {
+    pub const ChainForkVersions: ForkVersions = ForkVersions {
+        genesis: Fork {
+            version: [1, 1, 112, 0], // 0x01017000
+            epoch: 0,
+        },
+        altair: Fork {
+            version: [1, 0, 0, 0], // 0x01017000
+            epoch: 0,
+        },
+        bellatrix: Fork {
+            version: [1, 0, 0, 0], // 0x01017000
+            epoch: 0,
+        },
+        capella: Fork {
+            version: [1, 1, 112, 1], // 0x01017001
+            epoch: 256,
+        },
+        deneb: Fork {
+            version: [1, 1, 112, 2], // 0x01017002
```

Comment:
Instead of having a potentially wrong comment, just use the hex macro:

```suggestion
            version: hex!("01017002")
```

Reply:
isnt hex std?